packages/textinput/src/css/index.js

```diff
@@ -1,4 +1,12 @@
-import { colors, layout, type } from '@pluralsight/ps-design-system-core'
+import {
+  colorsBackgroundDark,
```

Comment:
It's really good that you're doing this to allow treeshaking at least parts of core.  I don't think I've been doing this.  We should call this out for a good refactor on these color conversion PRs if aren't seeing them, I think.  cc @EdwardIrby